clientjs/rxhtml.js

```diff
@@ -1998,6 +1998,15 @@ var RxHTML = (function () {
       window.scrollTo(0, 0);
       var state = { service: connection, data: null, view: fresh(where), current: "view" };
       self.__current = state;
+
+      const viewstate = localStorage.getItem('rxviewstate');
+      if (null !== viewstate) {
+        localStorage.removeItem('rxviewstate');
+        console.log(Object.assign({}, init));
+        Object.assign(init, JSON.parse(viewstate));
+        console.log(Object.assign({}, init));
```

Comment:
@mathgladiator I didn't have any better luck merging into `init` here. I know it isn't a deep merge, but the nested object doesn't exist in the intial `init`. Thoughts?

Reply:
I am using https://github.com/ymmyk/adamatests/blob/main/frontend/nametest.rx.html as my test. The goal for me is to have the buttons that have been pressed remain pressed after the reload